_posts/2018/2018-09-30-dont-loop-and-do-work-in-jobs.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Don’t Loop & Do Work in Jobs
+description: 'Loop and enqueue, then work on one object.'
+layout: article
+category: ruby
+image:
+  base: '2018/dont-loop-and-do-work-in-jobs'
+  alt: 'Fruit Loops'
+  source: 'https://unsplash.com/photos/chp1ITgplkA'
+  credit: 'Etienne Girardet'
+---
+
+Getting as much of the slow or non-essential work of your application into asynchronous jobs is a good idea for the overall performance of your application.
+
+## Instead of…
+
+...doing jobs that iterate over a group of objects, doing work on each one.
+
+```ruby
+class DoABunchOfTranslationsJob < ApplicationJob
+  def perform
+    Text.find_each do |text|
+      text.do_a_slow_translation
+    end
+  end
+end
+```
+
+
+## Use…
+
+...a job to enqueue a series of jobs that acts on every object individually.
```

Comment:
So, the main thing here is: oh, we're not iterating anymore, we're just running a series of operations on each object?

I had to read it a couple of times to get at what you were trying to tell me to do/what the main difference in this case would be.

Reply:
Reading the code example below, I think you need a better statement for line 32.